esports_data_from_rimble/Makefile

```diff
@@ -38,16 +38,14 @@ build: tmp/x.wasm
 .PHONY: match-winner
 match-winner: check build
 	@echo "Running WASM module..."
-	@sed -e 's/Rimble API key/$(RIMBLE_API_KEY)/' ./match-winner.json.template > ./tmp/match-winner.json
-	@cat ./tmp/match-winner.json | bky-as attest-fn-call >tmp/out.json
```

Comment:
The one garden tend-ish thing I left in was removing this templating bit from rimble example. Instead just let user modify the templated file by hand like the other examples

Reply:
Is this change required for the PR? I am worried that this is reaching beyond the goal of this PR and possibly introducing breaking changes to workflows/processes or documentation (for example, I think Mike uses the makefile run targets). 

IMO of we are going to make changes that are for example tidying, unifying patterns between examples, or changing the behavior of the examples/example makefiles that are not required for test automation, we should push those to another PR or eng agenda to deal with those changes separately.